platform-sdk/swirlds-common/src/main/java/com/swirlds/common/threading/IntakePipelineManager.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.common.threading;
```

Comment:
This seems an odd place to put this class. This should be platform internal code. Stuff in swirlds common is stuff that is intended for use by applications.

Reply:
Moved